targets/l2_switch/p4src/l2_switch.p4

```diff
@@ -1,5 +1,5 @@
 //uncomment to enable openflow
-#define OPENFLOW_ENABLE
+//#define OPENFLOW_ENABLE
```

Comment:
How will it break it? This flag is defined when compiling bm-p4ofagent and passed to the P4 compiler?